DSESwift/DSEConfig.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  DSEConfig.swift
+//  SensePlatform
```

Comment:
It comes from the project name in xcode. sense-ios-lib has the project name of SensePlatform.